src/dmd/astbase.d

```diff
@@ -4470,6 +4470,7 @@ struct ASTBase
         size_t len;         // number of code units
         ubyte sz = 1;       // 1: char, 2: wchar, 4: dchar
         char postfix = 0;   // 'c', 'w', 'd'
+        bool interpolate = false; // true if prefixed with 'i'
```

Comment:
Please make this a Ddoc comment. I know the other ones are not, but we're trying to improve. I would also flesh out the comment a bit, something like: `true if  an interpolated string, i.e. prefixed with 'i'`. Please also use backticks for code in comments.

Reply:
Should I also add ddoc comments to expression.d, or is that supposed to go away at some point?

Reply:
Hmm, that's a good question. I see that I added the comment to `astbase.d`. But I would say aim to add the comments in other files, i.e. not in `astbase.d`.